src/test/java/com/puppycrawl/tools/checkstyle/checks/SuppressWarningsHolderTest.java

```diff
@@ -517,6 +519,50 @@ public void testAliasList2() throws Exception {
                 expected);
     }
 
+    @Test
```

Comment:
Please add one more test case: a case where we have two check modules of the same check, but with one missing the `Check` suffix